tir/technologies/webapp_internal.py

```diff
@@ -805,7 +819,7 @@ def filling_date(self, shadow_root=None, container=None):
 
         click_type = 1
         base_date_value = ''
-        endtime = time.time() + self.config.time_out / 5
+        endtime = time.time() + self.config.time_out / 2
```

Comment:
On all these cases we are having timeout problems, that's why we divided by 2 instead of 5 to give more time.